client/src/components/PillIcon/PillIcon.stories.js

```diff
@@ -0,0 +1,7 @@
+import React, { Component } from "react";
+import { storiesOf } from "@storybook/react";
+import PillIcon from "./PillIcon";
+const text = "Los Angeles";
```

Comment:
There's no need for another variable here, just drop the text into the component